students/saleta_sebastian/lesson_02_flow_control/adding_factorials.py

```diff
@@ -0,0 +1,14 @@
+def get_sum_of_factorials():
+    input_number = int(input())
+    factorial = 1
+    sum = 0
```

Comment:
Avoid using of built-in variables names. Have a look at:
https://docs.python.org/3/library/functions.html